runtime/doc/debugger.txt

```diff
@@ -119,7 +119,6 @@ The following compile time preprocessor variables control the features:
     Debug Glyphs				FEAT_SIGNS
     Debug Highlights				FEAT_SIGNS
     Message Footer				FEAT_FOOTER
-    Balloon Evaluation				FEAT_BEVAL
```

Comment:
Yes, about these const : FEAT_SIGNS, FEAT_FOOTER and FEAT_BEVAL,  they only appears in this doc. 

I wasn't sure if I can delete all the ```2. Vim Compile Options					*debugger-compilation*``` paragraph . Should I do this in a commit ?